modules/loader/src/main/java/com/opengamma/strata/loader/CharSources.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2017 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.loader;
+
+import java.io.File;
+import java.nio.charset.Charset;
+import java.nio.file.Path;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.ByteSource;
+import com.google.common.io.CharSource;
+import com.google.common.io.Files;
+import com.opengamma.strata.collect.io.ResourceLocator;
+
+/**
+ * Helper that allows {@link CharSource} objects to be created.
+ */
+public final class CharSources {
```

Comment:
This class needs a private constructor. See `TestHelper.coverPrivateConstructor()` for tests